api-reference/v1.0/resources/groups_overview.md

```diff
@@ -0,0 +1,126 @@
+# Working with groups in Microsoft Graph
+
+Groups are collections of [users](https://developer.microsoft.com/en-us/graph/docs/api-reference/v1.0/resources/user) who share access to resources in Microsoft services or in your app. Microsoft Graph provides you with APIs to create and manage a variety of different types of groups and group functionality to suit your scenario needs. All operations in Microsoft Graph on groups require administrator consent.

```

Comment:
Added other principals to cover those cases. Also added a new column to capture supported member types